pallets/teerex/src/lib.rs

```diff
@@ -215,6 +224,13 @@ pub mod pallet {
 				sender,
 				block_hash
 			);
+
+			let sender_index = <EnclaveIndex<T>>::get(sender.clone());
+			let mut sender_enclave =
+				<EnclaveRegistry<T>>::get(sender_index).ok_or(Error::<T>::EmptyEnclaveRegistry)?;
+			sender_enclave.timestamp = <timestamp::Pallet<T>>::get().saturated_into();
+			<EnclaveRegistry<T>>::insert(sender_index, sender_enclave);
+
```

Comment:
`mutate` or `try_mutate` might be better but this is OK.

Can we include the timestamp check in the `confirm_processed_parentchain_block` unittest?

Reply:
“timestamp check“ added.